ani-cli

```diff
@@ -84,9 +84,8 @@ update_script() {
 dep_ch() {
 	for dep; do
 		if ! command -v "$dep" >/dev/null; then
-			if flatpak info io.mpv.Mpv 2>&1 | grep -q 'error'; then
-				die "Program \"$dep\" not found. Please install it."
-			fi
+			[ "$player_function" = "flatpak_mpv" ] && ! flatpak info io.mpv.Mpv 2>&1 | grep -q 'error' && continue
```

Comment:
Ive tried to run the code without flatpak (so without flatpak mpv) installed and it skipped it. Flatpak wont return message with "error" in it when it is not there.
```bash
+ printf '\33[2K\r\033[1;34mChecking dependencies...\033[0m\n'
Checking dependencies...
+ dep_ch wget sed grep
+ for dep in "$@"
+ command -v wget
+ for dep in "$@"
+ command -v sed
+ for dep in "$@"
+ command -v grep
+ '[' flatpak_mpv = download ']'
+ '[' flatpak_mpv = flatpak_mpv ']'
+ '[' flatpak_mpv '!=' debug ']'
+ '[' flatpak_mpv '!=' download ']'
+ dep_ch flatpak
+ for dep in "$@"
+ command -v flatpak
+ '[' flatpak_mpv = flatpak_mpv ']'
+ flatpak info io.mpv.Mpv
+ grep -q error
+ continue
+ dep_ch 'flatpak run io.mpv.Mpv'
+ for dep in "$@"
+ command -v 'flatpak run io.mpv.Mpv'
+ '[' flatpak_mpv = flatpak_mpv ']'
+ flatpak info io.mpv.Mpv
+ grep -q error
+ continue
+ case "$search" in
+ '[' -z '' ']'
+ printf 'Search anime\n>> '
Search anime
>> + read -r query
````

Reply:
See https://github.com/pystardust/ani-cli/pull/968#pullrequestreview-1241834889